docs/hugo/content/introduction/multitenant-deployment.md

```diff
@@ -3,16 +3,48 @@ title: Deploying Azure Service Operator v2 in multi-tenant mode
 linktitle: Multitenancy
 ---
 
-The current release version of the Azure Service Operator (ASO) deployment YAML installs the operator in _single-tenant_ mode:
-deployed in the `azureserviceoperator-system` namespace with _a single set of Azure credentials_,
-and manages resources in _any namespace_ in the cluster.
-That single operator deployment handles webhooks fired when Azure resources are changed.
+Currently, we support two types of multitenancy with Azure Service Operator (ASO): 
+
+### Single operator multitenancy (default)
+Single operator deployed in the `azureserviceoperator-system` namespace.
+This operator can be configured to manage resources with multiple different identities:
+* Single global credential `aso-controller-settings` deployed as part of operator deployment.
```

Comment:
Which namespace does this go into?

Reply:
Made it clear b stating ' in operator's namespace'